packages/config/config/devices/0x0299/1a90.json

```
@@ -0,0 +1,133 @@
+// Technisat series switches
+// Single-Switch
+{
+	"manufacturer": "TechniSat",
+	"manufacturerId": "0x0299",
+	"label": "Single-Switch",
+	"description": "Single-Switch",
+	"devices": [
+		{
+			"productType": "0x0002",
+			"productId": "0x1a90"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"supportsZWavePlus": true,
+	"paramInformation": {
+		"1": {
+			"label": "Central Scene notifications",
+			"description": "Setting this parameter to 0 will disable central scene notifications for multiple (2x-5x button presses)..",
```

```suggestion
			"label": "Central Scene: Multiple Button Presses",
```
seems more concise(?)

---

I used the other configs as an template whith that exact wording. But happy to change it if this is the way!

---

We're in the process of cleaning up old files, so that happens sometimes.